### ISSUE TITLE
DPL-419 - As an R&D scientist I would like a dedicated Hi-C library type this should be liked to a new request type so that we can have a dedicated pipeline to provide end to end tracking and the ability to report

### ISSUE THREAD

**Issue by stevieing** [sanger/limber#1108]
**User story**
As an R&D scientist I would like a dedicated Hi-C library type this should be linked to a new request type so that we can have a dedicated pipeline to provide end to end tracking and the ability to report

This should be:

- a copy of the standard 96 well PCR process as used in HTP except it should have a PCR norm plate generated as a child of the PCR_XP plate. 
- This PCR norm plate will replace the stock tubes. 
- The norm tubes with sequencing requests which are normally generated as child assets of the stock tubes should now be generated as child assets of the norm plate. 
- I would like the robot steps to have bed verification added as shown in the process map https://lucid.app/lucidchart/890fffbf-b9b7-49f9-9770-6ac2af982e67/edit?viewport_loc=-1568%2C2668%2C7797%2C4543%2CyMZkBuVBdHzy&invitationId=inv_3982f239-273b-4bce-b5d8-36e21add3563#.
-  In the process map you'll see sometimes we can skip the production of the norm plate and pool the libs directly from the PCR_XP plate, if this could also be supported by the LIMS that would be great but it's rare that this will happen so it's not essential.

**Who are the primary contacts for this story**
Ben

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] A dedicated Hi-C library type (Similar to the standard 96 well process used in HTP)
- [ ] A PCR norm plate generated as a child of the PCR_XP plate
- [ ] The PCR Norm plate will replace the stock tubes
- [ ] Norm tubes with sequencing requests should be generated as child assets of the norm plate
- [ ] Robot steps to have bed verification as shown in the process diagram
- [ ] Ability to skip production of the norm plate and pool the libs directly from the PCR_XP plate (Nice to have)

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**
